pkg/systemd/overview-cards/motdCard.scss

```diff
@@ -1,14 +1,3 @@
-#motd {
-  --motd--font-size: var(--pf-global--FontSize--sm);
-  background-color: transparent;
```

Comment:
This rule was actually intended

You can see the motd in http://localhost:9090/system after you add some  content in /etc/motd file.

Reply:
I just texted a bit interactively. THe idea is that you want o keep the looks of it exactly how it was. This heavily regressed. 
Please check this by yourself, if you manage to adjust the CSS to achieve the expected looks/ 

I can suggest that the only that can be removed is font-size setting as codeblock default font size is also small.

Reply:
When using `<CodeBlockCode>` directly, it looks identical to the old `<pre>`, let me know if this is OK.